src/System.Private.CoreLib/shared/System/Number.Formatting.cs

```diff
@@ -1205,9 +1205,7 @@ private static unsafe void UInt32ToNumber(uint value, ref NumberBuffer number)
         {
             while (--digits >= 0 || value != 0)
             {
-                // TODO https://github.com/dotnet/coreclr/issues/3439
-                uint newValue = value / 10;
-                *(--bufferEnd) = (byte)(value - (newValue * 10) + '0');
+                *(--bufferEnd) = (byte)(Math.DivRem(value, 10, out uint newValue) + '0');
```

Comment:
```suggestion
                *(--bufferEnd) = (byte)(Math.DivRem(value, 10, out value) + '0');
```

? Or would this regress the code quality?

Reply:
@jkotas Just checked - there is no difference in the codegen so I'll apply this suggestion